### PR TITLE
sentry: fix warnings being incorrectly logged

### LINKED ISSUE #44: sentry: python warnings logged with LOGGING_SENTRY_PYWARNINGS=False
Warnings are logged in Sentry even when `LOGGING_SENTRY_PYWARNINGS` is set to `False` and `logging.captureWarnings(True)` has been set.

Raven adds a `SentryHandler` to the root logger and since `py.warnings` is not among the excluded loggers, `logger.getLogger('py.warnings').propagate` is set to `True` which will execute the `SentryHandler` in the parent logger (root).

### PR DESCRIPTION
**Pull request by equadon**
Closes #44

### CODE REVIEW

File: tests/test_sentry.py [hunk 1]
```diff
@@ -84,6 +85,40 @@ def test_pywarnings(pywarnlogger):
     assert len(pywarnlogger.handlers) == 1
 
 
+def test_pywarnings_are_logged_once(pywarnlogger):
```
Comment (lnielsen) [inline tests/test_sentry.py]: Perhaps just one tiny improvement would be to factor out part of it as a fixture:

```
# conftest.py
@pytest.fixture()
def sentry_emit()
    from raven.handlers.logging import SentryHandler
    with patch.object(SentryHandler, 'emit') as sentry_emit:
        yield sentry_emit

#tests_....py
def test_pywarnings_are_logged_once(pywarnlogger, sentry_emit): 
    from invenio_logging.sentry import InvenioLoggingSentry
    # ...
```

The tests in the logging module do not make much use of fixtures though

File: setup.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
     ],
     'tests': tests_require,
     'sentry': [
-        'raven[flask]>=5.0.0,<5.5',
+        'raven[flask]>=6.0',
```
Comment (lnielsen) [inline setup.py]: What is the reason for this one? I have a vague recollection that there were some issues with this in the past.

**Comment by lnielsen** [comment]
All in all everything looks fine. Just one thing to clarify regarding the raven version

File: tests/test_sentry.py (hunk 1 above)
Comment (equadon) [reply tests/test_sentry.py]: Thank you! Yes, that's much cleaner. I'll change that.

File: setup.py (hunk 2 above)
Comment (equadon) [reply setup.py]: It was probably too much to upgrade to `>=6.0`. I actually only needed `>=5.10.2` to set the `logging_exclusions` but all it does is set `logging.getLogger('py.warnings').propagate = False` so we could actually skip upgrading at all and manually set `propagate`.

I also believe there was an incompatibility between Invenio-Logging's Raven version and Flask-Login which seems to have been fixed in `>=5.7.0`.

Is it fine if we change it to `5.10.2,<6` or do you know if there were any issues with those versions?

**Pull request merged**